src/SFA.DAS.EmployerAccounts.Web/ViewModels/AccountDashboardViewModel.cs

```diff
@@ -26,9 +26,12 @@ public class AccountDashboardViewModel
         public bool AgreementsToSign { get; set; }
         public int SignedAgreementCount { get; set; }
         public List<PendingAgreementsViewModel> PendingAgreements { get; set; }
-        public AccountDto AccountViewModel { get; set; }
+        public bool ApprenticeshipAdded { get; set; }
+        public bool ShowSearchBar { get; set; }
+        public bool ShowMostActiveLinks { get; set; }
+        public EAS.Portal.Types.Account AccountViewModel { get; set; }
         public Guid? RecentlyAddedReservationId { get; set; }
-        public IReservedFundingDto ReservedFundingToShow { get; set; }
-        public string ReservedFundingToShowLegalEntityName { get; set; }
+        public Reservation ReservedFundingToShow => AccountViewModel?.Organisations?.SelectMany(org => org.Reservations).FirstOrDefault(rf => rf.Id == RecentlyAddedReservationId) ?? AccountViewModel?.Organisations?.SelectMany(org => org.Reservations)?.LastOrDefault();
```

Comment:
`LastOrDefault` is not equivalent to latest, it's just the last `Reservation` returned by `SelectMany`.

Reply:
As we discussed, showing a single Reservation is only for MVS, and will be replaced by showing all reservations (and we don't store the created date of individual reservations - which we might actually want to do), and this improves on the existing behavior of just picking the first, if we don't find the supplied ReservationId (even if it doesn't _always_ return the latest).